### PR TITLE
Docs: service export type doc addition and related fixes

### PR DESCRIPTION
**Pull request by jenmalloy**

### CODE REVIEW

File: documentation/modules/ref-address-space-example-exports.adoc [hunk 1]
```diff
@@ -5,7 +5,14 @@
...
 
-This example shows the format used by the `ConfigMap` and `Secret` export types:
+You can export your address space information using the following three export types:
+* `ConfigMap`
+* `Secret`
+* `Service`
+
+== `ConfigMap` and `Secret` type export examples
+
+This example shows the format used by the `ConfigMap` export type. The format of the `Secret` export type uses the same keys as the `ConfigMap` export type, but the values are MD5 encoded.
```
Comment (k-wall) [inline documentation/modules/ref-address-space-example-exports.adoc]: My typo, sorry! "MD5" should read "Base64"

File: documentation/assemblies/assembly-address-space-examples.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@ include::../modules/ref-address-space-example.adoc[leveloffset=+1]
 
 include::../modules/ref-address-space-example-auth-service.adoc[leveloffset=+1]
 
-include::../modules/ref-address-space-example-exposing-endpoints.adoc[leveloffset=+1]
```
Comment (lulf) [inline documentation/assemblies/assembly-address-space-examples.adoc]: I dont think exposing example should be removed?

Comment (jenmalloy) [reply documentation/assemblies/assembly-address-space-examples.adoc]: @lulf Thanks for catching this.

File: documentation/modules/ref-address-space-example-exports.adoc (hunk 1 above)
Comment (jenmalloy) [reply documentation/modules/ref-address-space-example-exports.adoc]: @k-wall Fixed

File: documentation/modules/ref-address-space-example-exporting-endpoints.adoc [hunk 3]
```diff
@@ -28,7 +28,7 @@ spec:
       name: my-config <2>
 ----
 <1> (Required) The export type. The resulting `ConfigMap` contains the values in the format
```
Comment (laidan6000) [inline documentation/modules/ref-address-space-example-exporting-endpoints.adoc]: Suggested change:

(Required) The type of export, either `ConfigMap`, `Secret`, or `Service`.

File: documentation/modules/ref-address-space-example-exporting-endpoints.adoc [hunk 4]
```diff
@@ -28,7 +28,7 @@ spec:
...
-shown in link:link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-address-space-example-exports-messaging[example exports format]. For `Secret`, the same keys are used, but the values are base64-encoded. For
+shown in link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-address-space-example-exports-messaging[example exports format]. For `Secret`, the same keys are used, but the values are base64-encoded. For
 `Service`, a {KubePlatform} service of the type `ExternalName` is created. This provides applications running on {KubePlatform} with a way to
 inject endpoint information or provide a proxy service in the same namespace as the application. For more information see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-address-space-example-exports-messaging[example exports format].
 <2> (Required) The name of the resource to create and update.
```
Comment (laidan6000) [inline documentation/modules/ref-address-space-example-exporting-endpoints.adoc]: Should this be "create **or** update"?

File: documentation/modules/ref-address-space-example-exports.adoc [hunk 5]
```diff
@@ -5,7 +5,14 @@
 [id='ref-address-space-example-exports-{context}']
 = Address space example exports
 
-This example shows the format used by the `ConfigMap` and `Secret` export types:
+You can export your address space information using the following three export types:
+* `ConfigMap`
```
Comment (laidan6000) [inline documentation/modules/ref-address-space-example-exports.adoc]: This unordered list is not rendering in my Asciidoctor browser plug-in. The AsciiDoctor user manual suggests adding a blank line before the first list item but I'm not 100% sure this is required. See https://asciidoctor.org/docs/user-manual/#unordered-lists

**Pull request merged**